src/broker/Manager.cc

```diff
@@ -1140,11 +1140,6 @@ void Manager::DispatchMessage(const broker::topic& topic, broker::data msg)
 
 void Manager::Process()
 	{
-	// Ensure that time gets update before processing broker messages, or events
-	// based on them might get scheduled wrong.
-	if ( use_real_time )
```

Comment:
I can't say I'm 100% sure of the implications, with this changeset network time might lag a bit behind realtime. Events/timers scheduled from broker events will now be based on Zeek's actual notion of network time based on packets, rather than forcing time to be forwarded whenever a message is received.

For broker stores today, at least in the live realtime case, expiration should happen wherever the master store is located and that may have no relation to a workers notion of network time (and fast forwarding time doesn't help a lot here).

I do think that `allow_network_time_forward=F` should  set Broker's use_real_time false, too, or there needs to be a new option to control that separately. I'll add that, but we can iterate in follow-ups if it there's a need to control that separately.

Reply:
Makes sense! I am wondering whether it would be worth to add a metric to the telemetry framework that provides the difference between network time and wall clock. Probably only relevant if there are setups that need a large idle timeout.

Reply:
> For broker stores today, at least in the live realtime case, expiration should happen wherever the master store is located and that may have no relation to a workers notion of network time (and fast forwarding time doesn't help a lot here).

With all of these changes, how does master's time move forward?

Reply:
> Is this worked on or tracked anywhere?

IMO easiest to add via an external script as a gauge.